src/Tickets/Commerce/Order.php

```diff
@@ -195,6 +195,23 @@ public function register_post_type() {
 			'query_var'       => false,
 			'rewrite'         => false,
 			'capability_type' => 'post',
+			'capabilities'    => [
+				'edit_post'              => 'not-allowed-cap', // 'edit_post',
+				'read_post'              => 'read_post',
+				'delete_post'            => 'not-allowed-cap', // 'delete_post',
+				'edit_posts'             => 'edit_posts',
+				'edit_others_posts'      => 'edit_others_posts',
+				'delete_posts'           => 'not-allowed-cap', // 'delete_posts',
+				'publish_posts'          => 'not-allowed-cap', // 'publish_posts',
+				'read_private_posts'     => 'read_private_posts',
+				'read'                   => 'read',
+				'delete_private_posts'   => 'not-allowed-cap', // 'delete_private_posts',
+				'delete_published_posts' => 'not-allowed-cap', // 'delete_published_posts',
+				'delete_others_posts'    => 'not-allowed-cap', // 'delete_others_posts',
+				'edit_private_posts'     => 'not-allowed-cap', // 'edit_private_posts',
+				'edit_published_posts'   => 'not-allowed-cap', // 'edit_published_posts',
+				'create_posts'           => 'not-allowed-cap', // 'edit_posts',
```

Comment:
```suggestion
				'create_posts'           => 'not-allowed-cap', // 'create_posts',
```

Reply:
Actually, i think `create_posts` is mapped to `edit_posts` cap. I did a copy/paste from core caps so i think thats ok